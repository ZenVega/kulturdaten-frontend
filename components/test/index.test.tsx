import renderer from 'react-test-renderer';

import { Test } from '.';

test('Test renders', () => {
  const component = renderer.create(<Test />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
