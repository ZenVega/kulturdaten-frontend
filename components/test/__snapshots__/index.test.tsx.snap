// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test renders 1`] = `
<h2>
  Test
</h2>
`;
